elasticache/redis/Acornfile

```diff
@@ -0,0 +1,88 @@
+name:        "AWS Elasticache Redis"
+description: "Amazon's Redis compatible Elasticache cluster"
+
+args: {
+	// Name assigned to the cluster during creation. Default value is "RedisCluster".
+	clusterName: "RedisCluster"
+
+	// Key value pairs to apply to all resources.
+	tags: {}
+
+	// Deletion protection. Must be set to false in order to delete the Redis clutser. Default value is false.
+	deletionProtection: false
+}
+
+services: redis: {
```

Comment:
Should we just call it admin? since a user is likely to do:

```
services: redis: image: foo
containers: bar: {
  ...
   consumes["redis.admin"]
  ...
}
```
or something similar.

Reply:
Renamed to `admin` in the latest commit.